### DISCUSSION TITLE
摔在哪里 | /cn/2021/10/fall-place/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1538] [Comments]
快九个月大的丸子小公举
门牙已经长齐
穿着纱裙　扶着椅子　亭亭玉立
[&hellip;] 欲仗奶瓶走天涯
无奈还不会走只会爬
刚迈出腿　啪嗒&mdash;&mdash;
[&hellip;] 乖乖哟　小牙都磕破了小嘴皮
惊慌的妈妈闻声而至　十万火急
宝贝摔在哪里　摔在哪里
[&hellip;] 丸子倒是没哭没闹
只是满脸问号
为何爸爸给了一个长达半小时的抱抱
[&hellip;] 丸子第一次受伤的 &hellip;

https://yihui.org/cn/2021/10/fall-place/

**Comment by ZaiyanHere** [comment]
真的很少看到有父亲这样直接地表达「重重摔在了爸爸的心坎里」，很打动我。
我昨天和我爸爸说，我青春期的时候喜欢的男孩子都是和他风格完全不一样的，但是长大之后偏偏审美一百八十度大转变，开始被他这种「高标准严要求」的风格吸引，真是打脸，也希望我爸不要计较我之前对他的批评。我爸回答，「爸爸已经是老人了，什么都能放下，唯一放不下的是你的安全和幸福。」
不知道我小时候是不是也曾经摔在我爸的心坎里。

**Reply by yihui** [reply]
那肯定是曾经有过的。

另外：澄清一下丸子是我的干女儿哈，而我只是以干爹的旁观者角度写她的亲爹。我自家的儿子们小时候最多只能摔在我的耳边，而且有时候我还装听不见，哈哈。
